specs/Environment/NoiseLevelObserved/schema.json

```diff
@@ -45,5 +128,12 @@
       }
     }
   ],
-  "required": ["id", "type", "dateObservedFrom", "dateObservedTo", "location"]
+  "required": [
+    "id",
+    "type",
+    "dateObservedFrom",
+    "dateObservedTo",
+    "location",
+    "frequencies"
```

Comment:
be careful, -1 to this change. we cannot require `frequencies` to be provided always.